CONTRIBUTING.md

```diff
@@ -75,7 +75,7 @@ Designers who have agreed to protect the privacy of our clients may access our
 ### 2.1. Developer Setup and Documentation.
 
 To set up your development environment you'll need `ruby`, `node` and
-`postgresql`.
+`postgresql`. If on Windows using WSL2, please see [this document](docs/windows-wsl-dev-setup.md).
```

Comment:
Part of me thinks this should be a 

```markdown

> _For Windows users, see our [Windows Developer Setup](docs/windows-wsl-dev-setup.md)]

```

to set it apart from the main body; but I think it's fine as is and am merging !